tests/components/lupusec/test_config_flow.py

```diff
@@ -0,0 +1,94 @@
+""""Unit tests for the Lupusec config flow."""
```

Comment:
Other tests I am missing:
1. Import tests (the same but with different input as it came from YAML) (This also includes the places where the import can fail)
2. Duplicate config entry. So add a MockConfigEntry with data and then try to add the same one. It should abort because we already "configured" that one (Check the swiss public transport one for this)

Reply:
Done.